packages/pds/src/app-view/proxied/index.ts

```diff
@@ -471,6 +474,31 @@ export default function (server: Server, ctx: AppContext) {
     },
   })
 
+  server.app.bsky.feed.getFeed({
+    auth: ctx.accessVerifier,
+    handler: async ({ auth, params }) => {
+      const requester = auth.credentials.did
+      // @TODO cache the feedgen did lookup
+      const { data: feed } = await agent.api.app.bsky.feed
+        .getFeedGenerator({ feed: params.feed }, await headers(requester))
+        .catch((err) => {
+          console.log(err)
+          throw err
+        })
+      if (!feed.view.did) {
```

Comment:
We might want to address this with a lexicon change to make `did` required.

Reply:
+1 yup it shoulda been

Reply:
Logged this.